pkg/apiserver/apiserver.go

```diff
@@ -172,6 +141,7 @@ func InstallSupport(mux Mux, ws *restful.WebService) {
 	// TODO: convert healthz and metrics to restful and remove container arg
 	healthz.InstallHandler(mux)
 	mux.Handle("/metrics", prometheus.Handler())
+	mux.HandleFunc("/resetMetrics", metrics.Reset)
```

Comment:
Do we want to expose it in prod? Maybe we can do it only for debugging (similarly to /pprof? WDYT?